wiz-admission-controller/templates/cronjobmanager.yaml

```diff
@@ -0,0 +1,186 @@
+                - {{ . | toJson | quote }}
+              {{- end }}
+              env:
+              {{- if not .Values.wizApiToken.usePodCustomEnvironmentVariablesFile }}
+                - name: WIZ_CLIENT_ID
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.secretApiTokenName" . | trim }}
+                      key: clientId
+                      optional: false
+                - name: WIZ_CLIENT_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.secretApiTokenName" . | trim }}
+                      key: clientToken
+                      optional: false
+              {{- end }}
+                - name: WIZ_ENV
+                  value: {{ coalesce .Values.global.wizApiToken.clientEndpoint .Values.wizApiToken.clientEndpoint | quote }}
+              {{- if or .Values.global.httpProxyConfiguration.enabled .Values.httpProxyConfiguration.enabled }}
+                - name: HTTP_PROXY
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.proxySecretName" . | trim }}
+                      key: httpProxy
+                      optional: false
+                - name: HTTPS_PROXY
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.proxySecretName" . | trim }}
+                      key: httpsProxy
+                      optional: false
+                - name: NO_PROXY
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.proxySecretName" . | trim }}
+                      key: noProxyAddress
+                      optional: false
+              {{- end }}
+              {{- if .Values.logLevel }}
+                - name: LOG_LEVEL
+                  value: {{ .Values.logLevel }}
+              {{- end }}
+              {{- with .Values.podCustomEnvironmentVariables }}
+                {{- toYaml . | nindent 14 }}
+              {{- end }}
+              {{- with .Values.global.podCustomEnvironmentVariables }}
+                {{- toYaml . | nindent 14 }}
+              {{- end }}
+              {{- if .Values.podCustomEnvironmentVariablesFile }}
+                - name: CLI_ENV_FILE
+                  value: {{ .Values.podCustomEnvironmentVariablesFile }}
+                - name: USE_CLI_ENV_FILE
+                  value: "true"
+              {{- end }}
+                - name: WIZ_RUNTIME_METADATA_POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: WIZ_RUNTIME_METADATA_NODE_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: spec.nodeName
+                - name: K8S_NAMESPACE
```

Comment:
I think we might already have it from the projected service account token:
```
~ $ cat /var/run/secrets/kubernetes.io/serviceaccount/namespace
default~ $
```

Reply:
It's used in the kusto logger, I prefer to keep it like that